### PR TITLE
updated to Laravel Auditing version 7

### PR DESCRIPTION
**Pull request by zack-hable**
Removed support from Version 4 and switched to Version 7 (latest).  I'm not sure if the versions between would be supported (5-6).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,4 +73,16 @@ class SomeModel extends Model implements AuditableContract
...
+return [
+    // ...
+
+    'driver' => BetaPeak\Auditing\Drivers\FilesystemDriver::class,
+
+    // ...
+];
+```
+
+More information on using customer drivers with owen-it/laravel-auditing can be found on their [homepage](http://www.laravel-auditing.com/docs/7.0/audit-drivers)
```
Comment (ymslavov) [inline README.md]: I believe correct wording is "Community drivers"

**Review by ymslavov — CHANGES_REQUESTED** [review]

**Comment by ymslavov** [comment]
@zack-hable Thank you for your PR! There's just a minor wording tweak and I'll merge it in.

**Comment by zack-hable** [comment]
Awesome!  I have fixed that typo in the readme.

**Comment by zack-hable** [comment]
@ymslavov Are there any other issues?

**Comment by ymslavov** [comment]
Thank you for the contribution and sorry for the delay @zack-hable

**Pull request merged**